imqsauth.go

```diff
@@ -419,6 +651,60 @@ func setGroup(icentral *auth.ImqsCentral, groupName string, roles []string) bool
 	return true
 }
 
+// cleanGroup
+// Cycle through all users and remove the group from their permits
+func cleanGroup(ic *auth.ImqsCentral, groupIDString string) bool {
```

Comment:
So! From a previous conversation, on a previous PR where I mentioned that we need to provide the consumers of the authaus lib with the tools to better maintain the underlying DB.

I believe that we should create a new method on `authaus.Central` called `RemoveGroupFromAllUsers`. And it should pretty much contain the contents of this funciton (but return an error)